src/shared/Languages.ts

```diff
@@ -0,0 +1,73 @@
+export type LanguageKey =
```

Comment:
this doesnt need to be in shared anymore

Reply:
* we actually don't need this anymore now that we get the language key from vscode settings